superset-frontend/src/components/Chart/chartAction.js

```diff
@@ -389,6 +389,7 @@ export function exploreJSON(
       ownState,
     });
 
+    const datasourceInfo = formData.datasource.split('__');
```

Comment:
nit: deconstruct `datasource` then use it in for the split and the payload:
```
const { datasource } = formData;
// ...
logEvent(LOG_ACTIONS_LOAD_CHART, {
// ...
datasource,
datasource_id: datasourceInfo[0], 
datasource_type: datasourceInfo[1],
```

Reply:
changed!